diesel/src/expression/array_comparison.rs

```diff
@@ -13,7 +13,8 @@ pub struct In<T, U> {
 
 #[derive(Debug, Copy, Clone)]
 pub struct NotIn<T, U> {
-    in_expr: In<T, U>
+    left: T,
```

Comment:
I think it originally was because this code was based on the general infix expression code where the members are called `left` and `right`. I agree I should come up with a better name for this context.